lib/mix/prepare_assets.ex

```diff
@@ -112,19 +90,15 @@ defmodule Mix.Tasks.Antikythera.PrepareAssets do
       nil ->
         IO.puts("Skipping. Asset preparation is not configured.")
         IO.puts("Define `antikythera_prepare_assets` npm-script if you want antikythera to prepare assets for your gear.")
-      step ->
+      :npm_script ->
         install_packages!(env)
-        build_assets!(step, env)
+        build_assets!(env)
         dump_asset_file_paths()
     end
   end
 
-  defunp find_build_step_in_project(env :: v[String.t]) :: nil | :npm_script | :gulp do
-    cond do
-      npm_script_available?(env)  -> :npm_script
-      File.exists?("gulpfile.js") -> :gulp
-      true                        -> nil
-    end
+  defunp find_build_step_in_project(env :: v[String.t]) :: nil | :npm_script do
+    if npm_script_available?(env), do: :npm_script, else: nil
```

Comment:
Now we don't have to convert the boolean value returned by `npm_script_available?(env)` into `nil | :npm_script`.
Remove `find_build_step_in_project/1` and simply use `if-else` in `run_impl/1`.

Reply:
Fixed in 68a874a